build/building/multi-platform.md

```diff
@@ -0,0 +1,274 @@
+
+And inspect it:
+
+```console
+$ docker buildx inspect
+Name:   mybuilder
+Driver: docker-container
+
+Nodes:
+Name:      mybuilder0
+Endpoint:  unix:///var/run/docker.sock
+Status:    running
+Buildkit:  v0.10.4
+Platforms: linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
+```
+
+Now listing the existing builders again, we can see our new builder is
+registered:
+
+```console
+$ docker buildx ls
+NAME/NODE     DRIVER/ENDPOINT              STATUS   BUILDKIT PLATFORMS
+mybuilder     docker-container
+  mybuilder0  unix:///var/run/docker.sock  running  v0.10.4  linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
+default *     docker
+  default     default                      running  20.10.17 linux/amd64, linux/arm64, linux/arm/v7, linux/arm/v6
+```
+
+## Example
+
+Test the workflow to ensure you can build, push, and run multi-platform images.
+Create a simple example Dockerfile, build a couple of image variants, and push
+them to Docker Hub.
+
+The following example uses a single `Dockerfile` to build an Alpine image with
+cURL installed for multiple architectures:
+
+```dockerfile
+# syntax=docker/dockerfile:1
+FROM alpine:3.16
+RUN apk add curl
+```
+
+Build the Dockerfile with buildx, passing the list of architectures to
+build for:
+
+```console
+$ docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 -t username/demo:latest --push .
+...
+#16 exporting to image
+#16 exporting layers
+#16 exporting layers 0.5s done
+#16 exporting manifest sha256:71d7ecf3cd12d9a99e73ef448bf63ae12751fe3a436a007cb0969f0dc4184c8c 0.0s done
+#16 exporting config sha256:a26f329a501da9e07dd9cffd9623e49229c3bb67939775f936a0eb3059a3d045 0.0s done
+#16 exporting manifest sha256:5ba4ceea65579fdd1181dfa103cc437d8e19d87239683cf5040e633211387ccf 0.0s done
+#16 exporting config sha256:9fcc6de03066ac1482b830d5dd7395da781bb69fe8f9873e7f9b456d29a9517c 0.0s done
+#16 exporting manifest sha256:29666fb23261b1f77ca284b69f9212d69fe5b517392dbdd4870391b7defcc116 0.0s done
+#16 exporting config sha256:92cbd688027227473d76e705c32f2abc18569c5cfabd00addd2071e91473b2e4 0.0s done
+#16 exporting manifest list sha256:f3b552e65508d9203b46db507bb121f1b644e53a22f851185d8e53d873417c48 0.0s done
+#16 ...
+
+#17 [auth] username/demo:pull,push token for registry-1.docker.io
+#17 DONE 0.0s
+
+#16 exporting to image
+#16 pushing layers
+#16 pushing layers 3.6s done
+#16 pushing manifest for docker.io/username/demo:latest@sha256:f3b552e65508d9203b46db507bb121f1b644e53a22f851185d8e53d873417c48
+#16 pushing manifest for docker.io/username/demo:latest@sha256:f3b552e65508d9203b46db507bb121f1b644e53a22f851185d8e53d873417c48 1.4s done
+#16 DONE 5.6s
+```
+
+> **Note**
+> 
+> * `username` must be a valid Docker Hub username.
```

Comment:
I've started using "template" style text for things like this, see: https://github.com/docker/buildx/blob/master/docs/guides/drivers/docker-container.md?plain=1#L53. We could go for `<user>/<image>` or just `<image>` or something?

Not sure if we have a preferred style through the docs, but would be good to align on a single style for at least the build docs.

Reply:
Good point, I'm not sure myself about the templating format across our docs. cc @usha-mandya

Reply:
@jedevc @crazy-max Looked at a couple of examples within Docs. We've used `<username>/<image>`. Suggest we use the same across Build docs